java/src/jmri/NamedBeanBundle.properties

```diff
@@ -299,3 +299,22 @@ Automatic               = Auto
 rsType                  = Locomotive
 FunctionMomentary       = Momentary
 FunctionContinuous      = Continuous
+
+# color lists for jmri.util.ColorUtil and for panel loading.  Note that 
```

Comment:
I think you will find that defining these color names as public static fields in ColorUtil will be more optimal than relying on Bundle.getMessage() to get these values.